### DISCUSSION TITLE
Welcome to Fli Discussions!

### DISCUSSION THREAD

**Discussion by CaptnCodr** [CaptnCodr/Fli#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by shubhamkumar13** [comment]
so I am confused by this snippet in the Fli repo : https://github.com/CaptnCodr/Fli/blob/334b1bc12456bae53cc59db22dd23ed9434343a9/README.md?plain=1#L91
```fs
let log (output: string) = Debug.Log($"CLI log: {output}")

cli {
    Exec "dotnet"
    Arguments "--list-sdks"
    Output log
}
|> Command.execute
```
where does the `Debug` module come from?

**Reply by CaptnCodr** [reply]
There is a mistake, it should be `Debugger` from `System.Diagnostics`: 
https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.debugger?view=net-9.0

**Reply by shubhamkumar13** [reply]
Sorry for another stupid question 😅

I have a cli application that does log a lot of stuff when I'm using it in my console when I'm calling it from powershell.

I want to have the same stream of logs on my console when I use Fli's CE.

 Is that feasible? because even after updating to `Debugger.Log` I'm not seeing those logs.

**Reply by CaptnCodr** [reply]
> Sorry for another stupid question

There are no stupid questions. 😉 

There is actually an active PR (#80) that would handle your use case. Final details need to be tied up.

**Reply by shubhamkumar13** [reply]
Thanks! I got busy and forgot about this.

I like the tool and probably would love tinkering with it.

**Reply by CaptnCodr** [reply]
Hey @shubhamkumar13, 
you could try now, the referenced PR is released a quite time ago I wasn't aware anymore about this discussion. 

I hope this is solved now for you. 🙂
